framework/helpers/BaseHtml.php

```diff
@@ -1326,6 +1326,7 @@ public static function activeInput($type, $model, $attribute, $options = [])
         }
 
         static::setActivePlaceholder($model, $attribute, $options);
+        self::normalizeMaxLength($model, $attribute, $options);
```

Comment:
Seems that `normalizeMaxLength()` is now called twice for `activeTextInput()` and `activePasswordInput()`.

Reply:
@rob006 Yes indeed. Then you need to remove the normalization from `activeTextInput()`